src/customFileParser/unpackGeometry/Cone.ts

```diff
@@ -1,34 +1,33 @@
-import { RenderedGeometryGroups } from '../sharedFileParserTypes';
 import PropertyLoader from '../PropertyLoader';
 import * as THREE from 'three';
 
 const centerA = new THREE.Vector3();
 const centerB = new THREE.Vector3();
 
-function addOpenCone(groups: RenderedGeometryGroups, data: PropertyLoader) {
+function addOpenCone(groups: {[name: string]: any}, data: PropertyLoader) {
   centerA.copy(data.normal).multiplyScalar(data.height / 2).add(data.center);
   centerB.copy(data.normal).multiplyScalar(-data.height / 2).add(data.center);
-  groups.Cone.add(data.nodeId, data.treeIndex, data.color, centerA, centerB,
+  groups.Cone.add(data.nodeId, data.treeIndex, centerA, centerB,
   data.radiusA, data.radiusB, data.rotationAngle);
```

Comment:
Is adding space on this line good, so it is clear it is a continued line? This is coding style that I thought the linter would fix / do? What do you think?

Reply:
Fixed. For some reason the linter doesn't care -- will ask you about this.